### PR TITLE
front-end responsive pages

### PR DESCRIPTION
**Pull request by eileenabadi**

### CODE REVIEW

File: client/src/app/main/machine-profiles/machine-profiles.component.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="row">
+  <div class="profile col-md-8 col-lg-6 offset-sm-3">
```
Comment (jpastena) [inline client/src/app/main/machine-profiles/machine-profiles.component.html]: same here

File: client/src/app/main/upload/upload.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="row">
-  <div class="upload col-sm-4 offset-sm-3">
+  <div class="upload col-sm-9 offset-md-3">
```
Comment (jpastena) [inline client/src/app/main/upload/upload.component.html]: same here

File: client/src/app/login/login.component.scss [hunk 3]
```diff
@@ -29,10 +31,17 @@ small {
...
+  background-color: $colorWhite;
+  color: $colorIbm;
+  border: solid 1px $colorIbm;
+
+  &:hover {
+    cursor: pointer;
+    background-color: $colorIbm;
+    color: $colorWhite;
+  }
 }
```
Comment (mischaboldy) [inline client/src/app/login/login.component.scss]: can remove this styling if we add the primary button styling

File: client/src/app/main/machine-profiles/machine-profiles.component.scss [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  margin-top: 50px;
+}
+
+.top {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.search-input {
```
Comment (mischaboldy) [inline client/src/app/main/machine-profiles/machine-profiles.component.scss]: If we are going to use this searchbox on multiple places, can you make a _searchbox.scss file and put the styling in there? that way we only need it once

File: client/src/app/main/machine-profiles/machine-profiles.component.scss [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  color: $colorIbm;
+  border: solid 1px $colorIbm;
+  width: 120px;
+
+  &:hover {
+    cursor: pointer;
+    background-color: $colorIbm;
+    color: $colorWhite;
+  }
+}
```
Comment (mischaboldy) [inline client/src/app/main/machine-profiles/machine-profiles.component.scss]: can also become a btn primary button

File: client/src/app/main/upload/upload.component.scss [hunk 6]
```diff
@@ -23,19 +23,22 @@
...
+  background-color: $colorWhite;
+  color: $colorIbm;
+  border: solid 1px $colorIbm;
 
   &:hover {
-      cursor: pointer;
+    cursor: pointer;
+    background-color: $colorIbm;
+    color: $colorWhite;
   }
```
Comment (mischaboldy) [inline client/src/app/main/upload/upload.component.scss]: I think the upload button can also become a `btn primary-button`

File: client/src/app/login/login.component.html [hunk 7]
```diff
@@ -1,13 +1,13 @@
 <div class="container">
   <div class="row">
-    <div class="login-box col-md-4 offset-md-4">
+    <div class="login-box col-sm-4 offset-sm-4">
```
[the remark this answers is not in the document]
Comment (eileenabadi) [reply client/src/app/login/login.component.html]: it doesn't work this way.

File: client/src/app/login/login.component.html [hunk 8]
```diff
@@ -1,13 +1,13 @@
...
       <img class="ibm-image img-responsive" src="/assets/ibm-logo.svg">
       <span>Username or Email Address</span>
       <input class="input-field" type="text" placeholder="username or email address">
       <span>Password</span>
       <input class="input-field" type="password" placeholder="password">
       <small>Forgot Password ?</small>
-      <button class="submit-button btn btn-block">Submit</button>
+      <button class="submit-button">Submit</button>
```
[the remark this answers is not in the document]
Comment (eileenabadi) [reply client/src/app/login/login.component.html]: i'll change the button styling and remove unnecessary css after it merges

**Pull request merged**